src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -12,22 +15,22 @@ public void countIn(int in) {
 	}
 
 	public boolean callCheck() {
-			return total == -1;
+			return total == UNDERFLOW_LIMIT;
 		}
 
 	public boolean isFull() {
-				return total == 11;
+				return total == MAX_SIZE - 1;
 			}
-		
+
 	protected int peekaboo() {
 		if (callCheck())
-			return -1;
+			return UNDERFLOW_LIMIT;
```

Comment:
Zwracanie UNDERFLOW_LIMIT to nie najlepszy pomysł, przydałaby się inna stała albo zostawić -1